node/lib/vsotask.ts

```diff
@@ -408,7 +408,7 @@ export class TestPublisher {
 
     public testRunner: string;
 
-    public publish(resultFiles, mergeResults, platform, config) {
+    public publish(resultFiles, mergeResults, platform, config, runTitle, publishRunAttachments) {
         
         if(mergeResults == 'true') {
             _writeLine("Merging test results from multiple files to one test run is not supported on this version of build agent for OSX/Linux, each test result file will be published as a separate test run in VSO/TFS.");
```

Comment:
This task lib runs on windows agent as well so this line isn't always valid.  This should be removed.  If you want a warning on the xplat agent, it should be in the command handler of the xplat agent.  Can be addressed in a separate checkin.